CRM/Core/BAO/Setting.php

```diff
@@ -228,14 +228,29 @@ public static function getItem(
     $domainID = NULL
   ) {
 
+    $override_group = array();
```

Comment:
@sunilpawar Can you change this to $overrideGroup (so that we are following camelCase convention)